src/EPR.PRN.Backend.Obligation.UnitTests/Services/MaterialCalculationServiceTests.cs

```diff
@@ -1,41 +1,42 @@
 ﻿using EPR.PRN.Backend.Obligation.Services;
 using FluentAssertions;
+using System.Diagnostics.CodeAnalysis;
 
-namespace EPR.PRN.Backend.Obligation.UnitTests.Services
+namespace EPR.PRN.Backend.Obligation.UnitTests.Services;
+
+[ExcludeFromCodeCoverage]
```

Comment:
remove

Reply:
We don't need to look for code coverage on unit tests